internal/datacoord/meta.go

```diff
@@ -450,12 +450,13 @@ func (m *meta) GetQuotaInfo() *metricsinfo.DataCoordQuotaMetrics {
 	return info
 }
 
-// GetCollectionIndexFilesSize returns the total index files size of all segment for each collection.
-func (m *meta) GetCollectionIndexFilesSize() uint64 {
+// SetStoredIndexFileSizeMetric returns the total index files size of all segment for each collection.
+func (m *meta) SetStoredIndexFileSizeMetric() uint64 {
```

Comment:
no need to rename as SetXXX? since it returns a value.

Reply:
The primary function is to set values to metrics, so rename it from a 'get' to a 'set' method. The return value is intended solely for testing.